examples/parallel_memmap.py

```
@@ -38,44 +22,44 @@ def sum_row(input, output, i):
 
-        # Release the reference on the original in memory array and replace it
-        # by a reference to the memmap array so that the garbage collector can
-        # release the memory before forking. gc.collect() is internally called
-        # in Parallel just before forking.
-        samples = load(samples_name, mmap_mode='r')
+    # Release the reference on the original in memory array and replace it
+    # by a reference to the memmap array so that the garbage collector can
+    # release the memory before forking. gc.collect() is internally called
+    # in Parallel just before forking.
+    samples = load(samples_name, mmap_mode='r')
 
-        # Fork the worker processes to perform computation concurrently
-        Parallel(n_jobs=4)(delayed(sum_row)(samples, sums, i)
-                           for i in range(samples.shape[0]))
+    # Fork the worker processes to perform computation concurrently
+    Parallel(n_jobs=4)(delayed(sum_row)(samples, sums, i)
+                       for i in range(samples.shape[0]))
 
-        # Compare the results from the output buffer with the ground truth
-        print("Expected sums computed in the parent process:")
-        expected_result = samples.sum(axis=1)
-        print(expected_result)
+    # Compare the results from the output buffer with the ground truth
+    print("Expected sums computed in the parent process:")
+    expected_result = samples.sum(axis=1)
+    print(expected_result)
 
-        print("Actual sums computed by the worker processes:")
-        print(sums)
+    print("Actual sums computed by the worker processes:")
+    print(sums)
 
-        assert np.allclose(expected_result, sums)
-    finally:
-        try:
-            shutil.rmtree(folder)
-        except:
-            print("Failed to delete: " + folder)
+    assert np.allclose(expected_result, sums)
+finally:
+    try:
+        shutil.rmtree(folder)
+    except:  # noqa
+        print("Failed to delete: " + folder)
```

I think there is no point in hiding the exception if `shutil.rmtree(folder)` fails. Let's remove the try / except around it.